src/common/src/array/struct_array.rs

```diff
@@ -319,6 +314,23 @@ impl EstimateSize for StructArray {
     }
 }
 
+impl From<DataChunk> for StructArray {
+    fn from(chunk: DataChunk) -> Self {
```

Comment:
Can we make this conversion more explicit with a customized method?

Reply:
What's your consideration behind that? 👀
I think it is natural to have bidirectional conversion between DataChunk and StructArray. And [arrow](https://docs.rs/arrow-array/latest/arrow_array/array/struct.StructArray.html) also does it this way.

Reply:
Okay. That makes sense.